src/main/java/org/broadinstitute/hellbender/utils/IntervalUtils.java

```diff
@@ -53,7 +58,22 @@
      * </p>
      */
     public static final Comparator<Locatable> LEXICOGRAPHICAL_ORDER_COMPARATOR =
-            Comparator.comparing(Locatable::getContig,Comparator.nullsLast(String::compareTo))
+            Comparator.comparing(Locatable::getContig, Comparator.nullsLast(String::compareTo))
+                    .thenComparingInt(Locatable::getStart)
+                    .thenComparingInt(Locatable::getEnd);
+
+    /**
+     * Please see:  https://github.com/gpanther/java-nat-sort
+     *
+     * Also called alpha numerical sorting.
```

Comment:
Can you not use the order imposed by an available sequence dictionary and the existing `IntervalUtils.compareLocatables(first, second, dictionary)` method?